src/LoggerConfig.ts

```diff
@@ -9,7 +9,7 @@ import {HTMLLayoutColorTheme} from "./layouts/HTMLLayout";
 import {HTMLLayoutColors} from "./layouts/HTMLLayout";
 
 export default class LoggerConfig {
-    constructor(appender?: IAppender, private level: LogLevel = LogLevel.INFO, private tags?: string[]) {
+    constructor(appender?: IAppender, private level: LogLevel = LogLevel.INFO, private capture_stack: boolean = true, private tags?: string[]) {
```

Comment:
You should add new optional parameters to the end of constructor to prevent backward compatibility problems